.github/workflows/go.yml

```diff
@@ -39,8 +39,6 @@ jobs:
         uses: apache/skywalking-eyes@5c90c9698806040f50671a172a7cac17974ecebd
       - name: Update dependencies
         run: GOPROXY=https://proxy.golang.org go mod download
-      - name: Install Protoc
```

Comment:
If we remove this, we should install `protoc` in another way

Reply:
done